apis/bigquery/v1beta1/types.generated.go

```diff
@@ -622,151 +274,25 @@ type Dataset struct {
 }
 
 // +kcc:proto=google.cloud.bigquery.v2.DatasetReference
 type DatasetReference struct {
-	// Required. A unique ID for this dataset, without the project name. The ID
+	// Required. A unique Id for this dataset, without the project name. The Id
 	//  must contain only letters (a-z, A-Z), numbers (0-9), or underscores (_).
 	//  The maximum length is 1,024 characters.
-	DatasetId *string `json:"datasetId"`
-
-	// Required. The ID of the project containing this dataset.
-	ProjectId *string `json:"projectId"`
-}
-
-// +kcc:proto=google.cloud.bigquery.v2.DestinationTableProperties
-type DestinationTableProperties struct {
-	// Optional. Friendly name for the destination table. If the table already
-	//  exists, it should be same as the existing friendly name.
-	FriendlyName *string `json:"friendlyName,omitempty"`
-
-	// Optional. The description for the destination table.
-	//  This will only be used if the destination table is newly created.
-	//  If the table already exists and a value different than the current
-	//  description is provided, the job will fail.
-	Description *string `json:"description,omitempty"`
-
-	// Optional. The labels associated with this table. You can use these to
-	//  organize and group your tables. This will only be used if the destination
-	//  table is newly created. If the table already exists and labels are
-	//  different than the current labels are provided, the job will fail.
-	Labels map[string]string `json:"labels,omitempty"`
-}
-
-// +kcc:proto=google.cloud.bigquery.v2.DifferentialPrivacyPolicy
-type DifferentialPrivacyPolicy struct {
-	// Optional. The maximum epsilon value that a query can consume. If the
-	//  subscriber specifies epsilon as a parameter in a SELECT query, it must be
-	//  less than or equal to this value. The epsilon parameter controls the amount
-	//  of noise that is added to the groups — a higher epsilon means less noise.
-	MaxEpsilonPerQuery *float64 `json:"maxEpsilonPerQuery,omitempty"`
-
-	// Optional. The delta value that is used per query. Delta represents the
-	//  probability that any row will fail to be epsilon differentially private.
-	//  Indicates the risk associated with exposing aggregate rows in the result of
-	//  a query.
-	DeltaPerQuery *float64 `json:"deltaPerQuery,omitempty"`
-
-	// Optional. The maximum groups contributed value that is used per query.
-	//  Represents the maximum number of groups to which each protected entity can
-	//  contribute. Changing this value does not improve or worsen privacy. The
-	//  best value for accuracy and utility depends on the query and data.
-	MaxGroupsContributed *int64 `json:"maxGroupsContributed,omitempty"`
-
-	// Optional. The privacy unit column associated with this policy. Differential
-	//  privacy policies can only have one privacy unit column per data source
-	//  object (table, view).
-	PrivacyUnitColumn *string `json:"privacyUnitColumn,omitempty"`
+	DatasetId *string `json:"datasetId,omitempty"`
```

Comment:
For required fields, I believe we need to add `// +required`. I noticed that in the CRD, some fields that were previously marked as required are no longer required, which is likely not what we want.